kubernetes/docs/AdmissionregistrationApi.md

```diff
@@ -26,7 +26,7 @@ This endpoint does not need any parameter.
 
 ### HTTP request headers
 
- - **Content-Type**: application/json, application/yaml, application/vnd.kubernetes.protobuf
+ - **Content-Type**: Not defined
```

Comment:
why is this change? looks suspicious

Reply:
These docs are generated. I suspect the Swagger is broken, but I don't think it's worth fixing the swagger for this PR.